hw/mcu/stm/stm32_common/src/hal_gpio.c

```diff
@@ -479,8 +524,16 @@ int hal_gpio_init_out(int pin, int val)
     cfg.Pin = GPIO_MASK(pin);
     cfg.Mode = GPIO_MODE_OUTPUT_PP;
     cfg.Pull = GPIO_NOPULL;
+#if (defined GPIO_SPEED_FREQ_VERY_HIGH)
```

Comment:
while this should be OK, it would be better to write it the standard way: `#if defined(x)`